### PR TITLE
Codex 3 events

### PR DESCRIPTION
**Pull request by Dios-001**

### CODE REVIEW

File: src/components/Navbar/index.jsx
Comment (codexsit) [inline src/components/Navbar/index.jsx:1]: why is there a change in navbar component when u were working on events page/route

File: src/pages/Events/index.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const borderStyle = {
+    color: "transparent",
+    WebkitTextStrokeWidth: "0.5px",
+    WebkitTextStrokeColor: "white",
+  };
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    setUsers(userData);
+  }, []);
```
Comment (codexsit) [inline src/pages/Events/index.jsx]: no need to use `useEffect`, directly assign userData to useState

File: src/pages/Events/index.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import Card from "./card";
+import userData from "./events.json";
+
+function EventPage() {
+  const borderStyle = {
+    color: "transparent",
+    WebkitTextStrokeWidth: "0.5px",
+    WebkitTextStrokeColor: "white",
+  };
+  const [users, setUsers] = useState([]);
```
Comment (codexsit) [inline src/pages/Events/index.jsx]: Shouldn't this be events and not user?

**Review by codexsit — CHANGES_REQUESTED** [review]
Make necessary changes

File: src/components/Navbar/index.jsx
Comment (Dios-001) [reply src/components/Navbar/index.jsx:1]: my bad, assumed that everyone had to put their link into navbar component, apart from that on app.jsx i had made one change to routes/index.jsx, added route to events page, should i delete em?

**Pull request merged**